embedded/imu_driver/imu_message.hh

```diff
@@ -34,7 +37,8 @@ struct ImuMessage : Message {
           mag_utesla_x,
           mag_utesla_y,
           mag_utesla_z,
-          timestamp);
+          timestamp,
+          imu_id);
```

Comment:
it's a little confusing that this is also the i2c address - consider renaming this field to `i2c_address` for consistency?

Reply:
Was using the i2c address as a proxy for a serial number, I renamed the other